### ISSUE TITLE
compiler-rt fails to build on windows. with Cmake error "Fetching target triple from compiler "MSVC" is not implemented." when 64 bit host and architecture selected 

### ISSUE THREAD

**Issue by nizmogtr** [llvm/llvm-project#75991]
Building compiler-rt on windows fails with the following error:

CMake Error at cmake/Modules/CompilerRTMockLLVMCMakeConfig.cmake:60 (message):
  Fetching target triple from compiler "MSVC" is not implemented.
Call Stack (most recent call first):
  cmake/Modules/CompilerRTMockLLVMCMakeConfig.cmake:11 (compiler_rt_mock_llvm_cmake_config_set_target_triple)
  cmake/Modules/CompilerRTUtils.cmake:351 (compiler_rt_mock_llvm_cmake_config)
  CMakeLists.txt:85 (load_llvm_config)

-- Configuring incomplete, errors occurred!

**Comment by EugeneZelenko** [comment]
@nizmogtr: Could you please elaborate?

**Comment by nizmogtr** [comment]
I do not believe you can build compiler-rt with a non llvm/clang compiler. I can build compiler-rt if I use a llvm/clang compiler
